api/internal.go

```diff
@@ -141,7 +142,78 @@ func (s *PartyInfo) GetPartyInfo() {
 	}
 }
 
-func (s *PartyInfo) PollPartyInfo() {
+// GetPartyInfoGrpc requests PartyInfo data from all remote nodes this node is aware of. The data
+// provided in each response is applied to this node.
+func (s *PartyInfo) GetPartyInfoGrpc() {
```

Comment:
Can `GetPartyInfo` and `GetPartyInfoGrpc` be combined more? It looks like a fair amount of similar functionality between the two methods.